tests/utils/fixtures/index.ts

```diff
@@ -6,14 +6,21 @@ import { test as base, chromium } from '@playwright/test';
 import { getExtensionsData } from './utils/getExtensionsData';
 import { waitForExtensions } from './utils/waitForExtenssions';
 
+import { readFileSync } from 'fs';
+import { join } from 'path';
+
+const versionFile = readFileSync(join(process.cwd(), 'src/config/versions.json'), 'utf-8');
+const versions = JSON.parse(versionFile);
+
 export const test = base.extend<{
   context: BrowserContext;
   extensionId: string;
 }>({
   // biome-ignore lint/correctness/noEmptyPattern:
   context: async ({}, use) => {
     // download fuel wallet
-    const fuelPathExtension = await downloadFuel('0.16.6');
+    console.log("DOWNLOADING WALLET VERSION", versions.default.wallet)
```

Comment:
Do we want this log?

Reply:
yes, this just runs in the playwright tests